dcrtimed/dcrtimed.go

```diff
@@ -38,14 +38,15 @@ const (
 
 // DcrtimeStore application context.
 type DcrtimeStore struct {
-	backend    backend.Backend
-	cfg        *config
-	router     *mux.Router
-	ctx        context.Context
-	httpClient *http.Client
+	backend        backend.Backend
+	cfg            *config
+	router         *mux.Router
+	ctx            context.Context
+	httpClient     *http.Client
+	apiTokenLookup map[string]bool
 }
 
-func (d *DcrtimeStore) sendToBackend(w http.ResponseWriter, method, route, contentType, remoteAddr string, body *bytes.Reader) {
+func (d *DcrtimeStore) sendToBackend(w http.ResponseWriter, method string, route string, headers http.Header, remoteAddr string, body *bytes.Reader) {
```

Comment:
please explain this change.

Reply:
This change was originally made to pass an Authentication: Bearer/Basic header through the proxy without adding an extra parameter.

The headers can be whitelisted in `sendToBackend` or in each of the proxy* methods, but either way passing a map rather than a separate param per header seems cleaner. I'll revert this if the original signature is preferred.

Reply:
only the content-type of the header is used, so i would just pass that.  if multiple header fields were used in sendToBackend, it would be ok.